feat(dnsdist): Allow setting TTL in DNSQuestion:spoof (#17701)
PowerDNS/pdns: pdns/dnsdistdist/docs/reference/dq.rst

    :param string tail: The new data
    :returns: true if the operation succeeded, false otherwise

  .. method:: spoof(ip|ips|raw|raws [, typeForAny[, ttl]])

    .. versionchanged:: 1.9.0
      Optional parameter ``typeForAny`` added.

    .. versionchanged:: 2.2.0
      Optional parameter ``ttl`` added.

    Forge a response with the specified record data as raw bytes. If you specify list of raws (it is assumed they match the query type), all will get spoofed in.

    :param ComboAddress ip: The `ComboAddress` to be spoofed, e.g. `newCA("192.0.2.1")`.
    :param table ComboAddresses ips: The `ComboAddress`es to be spoofed, e.g. `{ newCA("192.0.2.1"), newCA("192.0.2.2") }`.
    :param string raw: The raw string to be spoofed, e.g. `"\\192\\000\\002\\001"`.
    :param table raws: The raw strings to be spoofed, e.g. `{ "\\192\\000\\002\\001", "\\192\\000\\002\\002" }`.
    :param int typeForAny: The type to use for raw responses when the requested type is ``ANY``, as using ``ANY`` for the type of the response record would not make sense.

>     :param int ttl: The TTL to set for the spoofed response, by default this is 60.

Reviewer: ```suggestion
    :param int ttl: The TTL to set for the spoofed response, with a default value of 60.
```
